pyproject.toml

```diff
@@ -23,7 +23,7 @@ dependencies = [
     "beautifulsoup4",
     "bleach",
     "defusedxml",
-    "entrypoints>=0.2.2",
+    "importlib-metadata;python_version<3.8",
```

Comment:
One thing: we have been trying to avoid `importlib-metadata<3.6`, and the stdlib one in `python<3.10`: it loads _every installed entrypoint_, which can take _a couple seconds_.

- https://github.com/jupyterlab/jupyterlab_server/blob/main/pyproject.toml#L27
- https://github.com/jupyterlab/jupyterlab_server/blob/main/jupyterlab_server/translation_utils.py#L20